app/lib/utils/string.js

```diff
@@ -0,0 +1,3 @@
+export function escapeShellArgument (text) {
```

Comment:
I think you're commenting on the wrong thing here. This is about shell arguments.